infra/modules/database/main.tf

```diff
@@ -75,11 +75,14 @@ resource "aws_rds_cluster_instance" "instance" {
   monitoring_interval                   = 30
   performance_insights_enabled          = true
   performance_insights_retention_period = 93
+
+  # checkov:skip=CKV_AWS_354:Ignore the managed customer KMS key requirement for now
```

Comment:
This is just... a common thing we say. I don't think we are ever going to get around to it. At least not here. But maybe in the template!